lib/cli/commands/executeswap.ts

```diff
@@ -0,0 +1,32 @@
+import { callback, loadXudClient } from '../command';
+import { Arguments } from 'yargs';
+import { ExecuteSwapRequest } from '../../proto/xudrpc_pb';
+
+export const command = 'executeswap <pair_id> <maker_order_id> <maker_peer_pub_key> <taker_order_id>';
+
+export const describe = 'execute a swap on between a maker peer order and a taker own order (nomatching mode only)';
```

Comment:
> This still needs to be addressed I believe. Let's just go with <pair_id> <order_id> <peer_pub_key> [quantity] as you've done below.

Sorry, forgot to change that. 

> As an aside, this might actually be a use case where I'd want that flat, not-grouped-by-peer, mapping of peer orders, so that the peer_pub_key wouldn't need to be specified on this call. But it's fine as is for now, just something to think about.

Yes, I was thinking of that too, but wasn't sure it's important enough for us, if that will be its only benefit. 

> With the recent change it'd make more sense to say something like "execute a swap on a peer order".

I left the `(nomatching mode only)` suffix, let me know if you want me to remove it.